.github/workflows/bors.yml

```diff
@@ -216,7 +204,6 @@ jobs:
       - name: find `#`-commands
         id: hash_commands
         run: |
-          chmod u+x scripts/lint_hash_commands.sh
```

Comment:
Let's revert this stray change (and simply land the rewrite of the hashCommandLinter instead).
```
         chmod u+x scripts/lint_hash_commands.sh